runtime/src/main/java/org/corfudb/util/serializer/JsonSerializer.java

```diff
@@ -85,7 +85,7 @@ public Object deserialize(ByteBuf b, CorfuRuntime rt) {
     public void serialize(Object o, ByteBuf b) {
         String className = o == null ? "null" : o.getClass().getName();
         if (className.endsWith(ICorfuSMR.CORFUSMR_SUFFIX)) {
-            className = "CorfuObject";
+            className = "SMRObject";
```

Comment:
I am not sure I understand why this change was made. In our case, CorfuObject serves as a simple annotation. In fact, the entire `CorfuObject` interface is marked as deprecated.

Reply:
Good catch, I actually didn't want to change this, intellij did it. 

In this context, it actually doesn't matter because there are problems to serializing deserializing SMRObjects, which @WenbinZhu  is removing.